.changeset/mighty-planets-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/parcel-transformer-markdown-html': minor
+---
+
+Add example transform for html code blocks that need to go full width when content section is expanded to full screen.
```

Comment:
was this a rebase change, perhaps you should rebase off latest `main`?

Reply:
i did rebase off latest main